### PR TITLE
Add persistent dir to oe-init/init-new-build oe-init/internal/gen-oe-…

### PR DESCRIPTION
**Pull request by williamodwyerdb**
…setup.py (speed up hashing)

### CODE REVIEW

File: oe-init/internal/gen-oe-setup.py [hunk 1]
```diff
@@ -7,13 +7,14 @@ def main():
 	parser = argparse.ArgumentParser(description='Generate an OE setup script')
 
 	parser.add_argument('--base-path', help='Path to base working directory', default=None)
-	parser.add_argument('--layers-path', help='Path to OE layers directory', default='openembedded'
-)
+	parser.add_argument('--layers-path', help='Path to OE layers directory', default='openembedded')
+
```
Comment (jackmitch) [inline oe-init/internal/gen-oe-setup.py:11]: drop this new line

File: oe-init/internal/gen-oe-setup.py [hunk 2]
```diff
@@ -97,6 +98,14 @@ def main():
 	ss_dir = os.path.abspath(ss_dir)
 	print_info("Selected shared state dir: " + ss_dir)
 
+	if args.pers_dir == "default":
+		pers_dir = os.path.join(root_path, "persistance")
```
Comment (jackmitch) [inline oe-init/internal/gen-oe-setup.py]: I think this should be kept as `cache` as is the default directory name when stored under the build directory

**Comment by jackmitch** [comment]
Looks good with those two minor changes, thanks!

File: oe-init/internal/gen-oe-setup.py (hunk 1 above)
Comment (williamodwyerdb) [reply oe-init/internal/gen-oe-setup.py:11]: There's no new line here, I just moved the round bracket back up to line #10

File: oe-init/internal/gen-oe-setup.py (hunk 2 above)
Comment (williamodwyerdb) [reply oe-init/internal/gen-oe-setup.py]: Updated in new commit

**Review by williamodwyerdb — COMMENTED** [review]
Changes made

**Pull request merged**